<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-AG2QBG6" failures="1" tests="2" name="cucumberOptions.TestRunner" time="428.302" errors="0" timestamp="2020-10-01T14:38:56 CEST" skipped="0">
  <testcase classname="cucumberOptions.TestRunner" name="runScenario" time="394.544"/>
  <system-out/>
  <testcase classname="cucumberOptions.TestRunner" name="runScenario" time="33.758">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at stepDefinitions.LoginTests.homepage_is_displayed(LoginTests.java:70)
at âœ½.Homepage is displayed(file:src/test/java/features/Login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- cucumberOptions.TestRunner -->
